apache/trafficcontrol
# cachegroupparameters rewrite

## What does this PR (Pull Request) do?

Rewrites /cachegroupparameters in go
- [x] This PR fixes #3809 


## Which Traffic Control components are affected by this PR?


- Documentation
- Traffic Ops

## What is the best way to verify this PR?
- Verify that a GET to /cachegroupparameters returns a list of all of the cachegroup / parameter associations
- Verify that a PUT to /cachegroupparameters with only one association like so:
{"cachegroupId":1, "parameterId":2}
adds the association to the table and appears with a GET
- Verify that a PUT to /cachegroupparameters with multiple associations like so:
[{"cachegroupId":1, "parameterId":2}, {"cachegroupId":1, "parameterId":2}, {"cachegroupId":2, "parameterId":2}]
adds all of the associations to the table and they all appear with a GET

## If this is a bug fix, what versions of Traffic Control are affected?



## The following criteria are ALL met by this PR




## Additional Information
